src/js/core.js

```diff
@@ -1145,6 +1145,15 @@
             }
         },
 
+        getContent: function (index) {
+            index = index || 0;
+
+            if (this.elements[index]) {
+                return this.elements[index].innerHTML.trim();
```

Comment:
I was about to suggest using `.serialize().value` instead but regarding `serialize()` it seems that it serialize all referenced MediumEditor instance since it loops on all `this.element`. Am I right?